src/components/events/create-event/advanced-options/Ingredient.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const Ingredient = ({ item, removeIngredient }) => {
+  return (
+    <div
+      key={item.id}
+      onClick={() => removeIngredient(item.id)}
+      style={{
+        display: "flex",
+        alignItems: "center",
+        background: "#FFBF1A",
+        opacity: "0.75",
+        borderRadius: "10px",
+        marginTop: "20px",
+        marginRight: "20px",
+        marginBottom: "20px",
+        color: "white",
+        fontSize: "1.6rem",
+        width: "auto",
+        maxWidth: "400px",
+        whiteSpace: "pre-line",
+        padding: "12px 20px",
+        cursor: "pointer",
```

Comment:
would all these extensive in-line styles be better to have stored either in css or as an imported style via /styles/index.js? I havent looked too closely at each one, but some of them seem to be styled either similarly or have almost identical attribute assignments, so it may make code easier to read and DRYer to separate it out.